templates/tag/problem.html

```diff
@@ -96,6 +96,9 @@ <h2>{{ content_title }}</h2>
                 <th>Assigner</th>
             </tr>
         <thead>
+        <div class="alert alert-warning alert-dismissable">
+            {{_("If your problem tag is false, you can comment below or use report button on the top of the right")}}
```

Comment:
Nah, something like: the problem tag might not be correct. Please feel free to leave a comment or use the report button at the top right of the navigation bar.